qcodes/instrument_drivers/AlazarTech/ATS.py

```diff
@@ -396,10 +431,26 @@ def config(self, clock_source=None, sample_rate=None, clock_edge=None,
                              external_trigger_range)
         self._set_if_present('trigger_delay', trigger_delay)
         self._set_if_present('timeout_ticks', timeout_ticks)
+        self._set_if_present('aux_io_mode', aux_io_mode)
+        self._set_if_present('aux_io_param', aux_io_param)
         # endregion
 
+        # handle that external clock and internal clock uses
+        # two different ways of setting the sample rate.
+        # We use the matching one and mark the order one
+        # as up to date since it's not being pushed to
+        # the instrument at any time and is never used
+        if clock_source == 'EXTERNAL_CLOCK_10MHz_REF':
```

Comment:
why do you not put a call to `self.get_sample_rate` here?

Reply:
Because that would include the decimation which we don't want here but I have changed the code to make the decimation correction optional instead.